### DISCUSSION TITLE
Hourly Total Site Energy from Energy Plus

### DISCUSSION THREAD

**Discussion by gtmlou** [CenterForTheBuiltEnvironment/pythermalcomfort#91] [General]
I asked in IDF for the hourly Total Side Energy as below:                                                                                                                 Output: Variable,
    *,                       !- Key Value
    Total Site Energy,       !- Variable Name
    Hourly;                  !- Reporting Frequency                                                                                                                                                                    But I didn't get any values in the simulation outputs. Why? Does anyone know what I should do? Thnks

**Comment by FedericoTartarini** [comment]
Hi, sorry but I don't understand your question. Is your question related to pythermalcomfort or energyplus?

**Reply by gtmlou** [reply]
It is related mainly to energyplus. Actually, I have the 8760 hourly values of pmv and ppd for my model and I want energyplus to give me the corresponding hourly total site energy but although I put in the IDF as required output variable the below:
*, !- Key Value
Total Site Energy, !- Variable Name
Hourly; !- Reporting Frequency
I don't get any values in the simulation outputs.

**Reply by FedericoTartarini** [reply]
I am not understanding then why you opened a discussion here since this is the pythermalcomfort repository
